src/config/prisma/prisma-client.ts

```diff
@@ -15,39 +15,40 @@ export async function startORM() {
     // await prisma.user.deleteMany()
 
     if (process.argv[process.argv.length - 1] === 'seed') {
-      await prisma.user.create({
-        data: {
-          ...userData,
-          venue: {
-            create: {
-              ...venueData,
-              location: {
-                create: locationData,
-              },
-              activities: {
-                create: [
-                  {
-                    ...activityData,
-                    location: {
-                      create: locationData,
-                    },
-                  },
-                ],
+      for (let i=0; i<userData.length; i++){
+        const user = await prisma.user.create({
+          data: {
+            ...userData[i],
+            venue: {
+              create: {
+                ...venueData[i],
+                location: {
+                  create: locationData[i],
+                },
               },
             },
           },
-        },
-        include: {
-          venue: {
-            include: {
-              activities: true,
+          include: {
+            venue: {
             },
           },
-        },
-      })
-
-      // await prisma.$disconnect()
-      // shutDown('database successful seeded')
```

Comment:
Add this code back and running please.

Reply:
Its added back in. Resolving the conversation for this.